internal/errors/code.go

```diff
@@ -22,6 +22,8 @@ const (
 
 	// General function errors are reserved Codes 100-999
 	InvalidParameter Code = 100 // InvalidParameter represents and invalid parameter for an operation.
+	InvalidAddress   Code = 101
```

Comment:
Note to reviewers I add these are non-generic InvalidParameters because I assume we want to return 400 in these two cases, please confirm?